### PR TITLE
Add asset credits in-game

### LINKED ISSUE #73: Specify asset licenses
This could be an own button in the menu

### PR DESCRIPTION
**Pull request by janhohenheim**
Resolves #73

image

### CODE REVIEW

File: src/screen/credits.rs [hunk 1]
```diff
@@ -27,9 +27,15 @@ fn enter_credits(mut commands: Commands) {
         .ui_root()
         .insert(StateScoped(Screen::Credits))
         .with_children(|children| {
+            children.header("Made by");
             children.label("Alice - Foo");
             children.label("Bob - Bar");
 
+            children.header("Assets");
+            children.label("Bevy logo - CC0 by Carter Anderson");
```
Comment (MiniaczQ) [inline src/screen/credits.rs:35]: Isn't it Bevy Org now?

**Review by MiniaczQ — APPROVED** [review]

File: src/screen/credits.rs (hunk 1 above)
Comment (janhohenheim) [reply src/screen/credits.rs:35]: Fair enough

File: src/ui_tools/widgets.rs [hunk 2]
```diff
@@ -5,8 +5,8 @@ use bevy::{ecs::system::EntityCommands, prelude::*, ui::Val::*};
 use super::{
     interaction::InteractionPalette,
     palette::{
-        BACKGROUND, BUTTON_HOVERED_BACKGROUND, BUTTON_PRESSED_BACKGROUND, BUTTON_TEXT, LABEL_TEXT,
-        NODE_BACKGROUND,
+        BACKGROUND, BUTTON_HOVERED_BACKGROUND, BUTTON_PRESSED_BACKGROUND, BUTTON_TEXT, HEADER_TEXT,
+        LABEL_TEXT, NODE_BACKGROUND,
     },
```
Comment (benfrankel) [inline src/ui_tools/widgets.rs:10]:
```suggestion
palette::*,
```

**Review by benfrankel — CHANGES_REQUESTED** [review]

File: src/ui_tools/widgets.rs (hunk 2 above)
Comment (janhohenheim) [reply src/ui_tools/widgets.rs]: [`937d2b5` (#123)](https://github.com/TheBevyFlock/bevy-template/pull/123/commits/937d2b54ef2e2d1daa6491bcf2df0ed6dee6d2fc)

**Pull request merged**